interfaces/le_ulpm.api

```diff
@@ -96,6 +96,22 @@ ENUM GpioState
     GPIO_OFF,              ///< Gpio off.
 };
 
+//--------------------------------------------------------------------------------------------------
+/**
+ * Possible ULPS configurations to select before shutdown. Value 3 is skipped as it should not be
+ * used according to swimcu_pm documentation.
+ */
+//--------------------------------------------------------------------------------------------------
+ENUM ULPSConfiguration
```

Comment:
I'm concerned that this is a bit swimcu_pm specific, but I'll let other comments on it.